### PR TITLE
Mapi 1084 atom

### PR DESCRIPTION
**Pull request by alexduf**
This PR focuses on the XML response parsing.

### CODE REVIEW

File: backup/src/main/scala/batches/Backup.scala [hunk 1]
```diff
@@ -40,9 +40,7 @@ class Backup @Inject() (conf: Configuration, ws: WSClient)(implicit ec: Executio
 
     val hubClient = new NotificationHubClient(conf.notificationHubConnection, ws)
     hubClient.submitNotificationHubJob(notificationJob).map {
-      case \/-(result) => result.items.foreach { job =>
-        Logger.info(s"Success, job created with id ${job.jobId}\n$job")
-      }
+      case \/-(job) => Logger.info(s"Success, job created with id ${job.jobId}\n$job")
```
Comment (davidfurey) [inline backup/src/main/scala/batches/Backup.scala]: Do you really want a newline character in a log message?

File: common/src/main/scala/gu/msnotifications/Responses.scala [hunk 2]
```diff
@@ -108,6 +108,26 @@ case class RegistrationResponse(registration: WNSRegistrationId, tags: List[Stri
...
+
+  implicit def reader[T](implicit reader: XmlReads[T]): XmlReads[AtomEntry[T]] = new XmlReads[AtomEntry[T]] {
+    def reads(xml: Elem) =  for {
+      title <- xml.textNode("title")
+      item <- getItem(xml)(reader)
+    } yield AtomEntry(title, item)
+  }
+
+  private def getItem[T](xml: Elem)(implicit reader: XmlReads[T]): HubResult[T] = {
+    val results = (xml \ "content").flatMap(_.child).collect {
```
Comment (davidfurey) [inline common/src/main/scala/gu/msnotifications/Responses.scala]: Could you use collectFirst instead?

Comment (alexduf) [inline common/src/main/scala/gu/msnotifications/Responses.scala]: Never used collectFirst before, sounds like a good idea!

**Comment by davidfurey** [comment]
:+1:

**Pull request merged**